expression/expression.go

```diff
@@ -1089,6 +1089,12 @@ func scalarExprSupportedByFlash(function *ScalarFunction) bool {
 		default:
 			return false
 		}
+	case ast.Upper, ast.Ucase:
+		switch function.Function.PbCode() {
+		case tipb.ScalarFuncSig_UpperUTF8:
```

Comment:
Why `upper` has two sig `ScalarFuncSig_UpperUTF8/ScalarFuncSig_Upper` and `lower` only has one sig `ScalarFuncSig_Lower`?